node/_crypto/randomFill_test.ts

```diff
@@ -0,0 +1,44 @@
+import { Buffer } from "../buffer.ts";
+import randomFill, { randomFillSync } from "./randomFill.ts";
+import { assertEquals, assertThrows } from "../../testing/asserts.ts";
+
+const validateNonZero = (buf: Buffer) => buf.some((ch) => ch > 0);
```

Comment:
This function returns a boolean, and it doesn't throw. So `validateNonZero(buf)` doesn't check whether buf actually includes non zero value or not.

Reply:
Updated.